crane/config.go

```diff
@@ -219,6 +227,20 @@ func (c *config) initialize() {
 	}
 }
 
+// Override configuration
+func (c *config) overrideWithGlobalFlags() {
+	// At the moment, only `--tag` flag triggers override
+	if len(c.Tag()) == 0 {
+		return
+	}
+
+	for _, container := range c.containerMap {
+		container.OverrideImageTag(c.Tag())
+	}
+
+	os.Setenv("CRANE_TAG_FLAG", c.Tag())
```

Comment:
I think we should not set this here. Instead, we can have kingpin read the value of `tag` from `CRANE_TAG`, which can then be used in hooks too. What do you think?